roadmap/implementers-guide/src/node/disputes/dispute-coordinator.md

```diff
@@ -380,13 +380,108 @@ candidates for approval votes a similar argument holds (if they come from
 approval-voting), but we also don't import them until a dispute already
 concluded. For actual dispute votes, we need two opposing votes, so there must be
 an explicit `invalid` vote in the import. Only a third of the validators can be
-malicious, so spam disk usage is limited to ```2*vote_size*n/3*NUM_SPAM_SLOTS```, with
-n being the number of validators.
+malicious, so spam disk usage is limited to `2*vote_size*n/3*NUM_SPAM_SLOTS`, with
+`n` being the number of validators.
 -
 More reasoning behind spam considerations can be found on
 this sr-lab ticket: https://github.com/paritytech/srlabs_findings/issues/179
```

Comment:
This references a private repo issue. I don't think it adds much value to this _public_ doc..